CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Applies new local tokens to `Link` ([#19](https://github.com/vtex-sites/gatsby.store/pull/19))
```

Comment:
IMO it should be in the `Changed` section because we are just applying some changes for components that already exist, WDYT?

Reply:
In fact we are adding new local tokens (mapping to old ones, but still). I think we can move to `Added` section too, because we did the same to the previous components (`Badge`, `Hero`, `Quantity Selector`, ....) 🤔